moviesurfer/data/download_data.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+import zipfile
+
+import wget
+
+logging.basicConfig(format="", level=logging.INFO)
```

Comment:
Previous comment was partially about this line. We usually define logging config in an actual script where we run main. In submodules of a library we just get logger using the command you wrote below. Otherwise, you override an existing cofig